CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.1.0] 2018-01-21
```

Comment:
Please remove the versioning as we do this post acceptance to reduce the number of times a submitter has to rebase.